### PR TITLE
Remove examples.

### PR DESCRIPTION
**Pull request by pv101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,6 @@ systems on mixed unstructured grids containing various element types. It is
 also designed to target a range of hardware platforms via use of an in-built
 domain specific language derived from the Mako templating engine.
 
-## Examples
```
Comment (FreddieWitherden) [inline README.md:12]: Keep heading and have text point to the examples repo.

File: doc/src/examples.rst [hunk 2]
```diff
@@ -21,11 +22,11 @@ simulation on a structured mesh:
 1. Create a working directory called ``euler_vortex_2d/``
 
 2. Copy the configuration file
-   ``PyFR/examples/euler_vortex_2d/euler_vortex_2d.ini`` into
```
Comment (FreddieWitherden) [inline doc/src/examples.rst:24]: I renamed some of the folders and meshes (check for .gz as well).

File: doc/src/examples.rst [hunk 3]
```diff
@@ -21,11 +22,11 @@ simulation on a structured mesh:
 1. Create a working directory called ``euler_vortex_2d/``
```
Comment (FreddieWitherden) [inline doc/src/examples.rst:22]: Do we need to bother creating a new directory?  Just seems like an extra step.

File: doc/src/examples.rst [hunk 4]
```diff
@@ -18,40 +19,34 @@ Euler Equations
...
 
-4. Run pyfr to convert the `Gmsh <http:http://geuz.org/gmsh/>`_
-   mesh file into a PyFR mesh file called ``euler_vortex_2d.pyfrm``::
+        pyfr import 2d-euler-vortex.msh 2d-euler-vortex.pyfrm
 
-        pyfr import euler_vortex_2d.msh euler_vortex_2d.pyfrm
+3. Run pyfr to partition the PyFR mesh file into two pieces::
 
-5. Run pyfr to partition the PyFR mesh file into two pieces::
+        pyfr partition 2 2d-euler-vortex.pyfrm .
```
Comment (FreddieWitherden) [inline doc/src/examples.rst:33]: I think some of these names are still wrong compared with what we have in the test case repo.

Comment (pv101) [reply doc/src/examples.rst:33]: I built the docs and copy pasted all commands into the latest version of the tests repo and it all ran fine. So I think it is all good. I'll merge.

**Pull request merged**